src/server/api/controllers/crops.controller.js

```diff
@@ -0,0 +1,13 @@
+const knex = require('../../config/db');
+
+const getAllCrops = async () => {
+  try {
+    return await knex('crops').select('crops.name', 'crops.plant_variety');
```

Comment:
FUN FACT: you don't need to do both return AND await. It should resolve to the same thing 🚀